pandas/plotting/_misc.py

```diff
@@ -323,20 +323,49 @@ def f(t):
 
 
 def bootstrap_plot(series, fig=None, size=50, samples=500, **kwds):
-    """Bootstrap plot.
+    """
+    Bootstrap plot on mean, median and mid-range statistics.
+
+    The bootstrap plot is used to estimate the uncertainty of a statistic
+    by relaying on random sampling with replacement. This function will
+    generate bootstrapping plots for mean, median and mid-range statistics
+    for the given number of samples of the given size.
 
     Parameters
     ----------
-    series: Time series
-    fig: matplotlib figure object, optional
-    size: number of data points to consider during each sampling
-    samples: number of times the bootstrap procedure is performed
-    kwds: optional keyword arguments for plotting commands, must be accepted
-        by both hist and plot
+
+    series : pandas.Series
+        Pandas Series from where to get the samplings for the bootstrapping.
+    fig : matplotlib.figure.Figure, default None
+        If given, it will use the `fig` reference for plotting instead of
+        creating a new one with default parameters.
+    size : int, default 50
+        Number of data points to consider during each sampling. It must be
+        greater or equal than the length of the `series`.
+    samples : int, default 500
+        Number of times the bootstrap procedure is performed.
+    kwds : keywords
```

Comment:
dict, I think this needs to be **kwds. @TomAugspurger

Reply:
Yeah we're doing `**kwds` with no type (it's not really a dict, but an unpacked dict).

Sorry @aydevosotros you can ignore the validation script here. Haven't updated it yet.